website/views.py

```diff
@@ -244,7 +244,11 @@ def validate_page_num(page, pages):
 
 
 def paginate(items, total, page, size):
-    pages = math.ceil(total / float(size))
+    try:
+        pages = math.ceil(total / float(size))
```

Comment:
I don't think this try and except is the right way to go. You should instead do a check if size is 0 then add some code to handle that specific situation.